tests/pyreverse/test_writer.py

```diff
@@ -87,6 +89,15 @@ def setup_colorized_dot(
     yield from _setup(project, colorized_dot_config, writer)
 
 
+@pytest.fixture()
+def setup_no_standalone_dot(
+    no_standalone_dot_config: PyreverseConfig, get_project: GetProjectCallable
+) -> Iterator[None]:
+    writer = DiagramWriter(no_standalone_dot_config)
+    project = get_project(TEST_DATA_DIR)
```

Comment:
You have to specify `name` here like above in `setup_colorized_dot`. Also you have to rename the files from `no_standalone_classes.dot` to `classes_no_standalone.dot` (same for package diagram).